src/views/Settings.vue

```diff
@@ -61,6 +61,7 @@
                   />
                   <amb-list-item
                     :title="version"
+                    id="version-element"
                     subtitle="Release Version"
```

Comment:
@vickywane let's update the text from "Release Version" to "Edge Software Version"